src/WebJobs.Script/Workers/Rpc/Configuration/LanguageWorkerOptionsSetup.cs

```diff
@@ -25,6 +26,13 @@ public LanguageWorkerOptionsSetup(IConfiguration configuration, ILoggerFactory l
 
         public void Configure(LanguageWorkerOptions options)
         {
+            string workerRuntime = _environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName);
+            if (!string.IsNullOrEmpty(workerRuntime) && workerRuntime.Equals(RpcWorkerConstants.DotNetLanguageWorkerName, System.StringComparison.OrdinalIgnoreCase))
```

Comment:
Don't we default to dotnet when the env variable is null or empty? should the logic be if it's null or empty or if it's set to dotnet?

Reply:
if null or empty - we need to parse the worker config files to be able to determine  language for each function - this is true for any language - for example a function app can have JS functions but did not set worker runtime